app/assets/stylesheets/comments.scss

```diff
@@ -0,0 +1,23 @@
+.row.comment {
+  border: 1px solid grey;
+  margin: 15px;
+  padding: 15px;
+
+  h5{
+    font-size: 25px;
+    font-weight: bold;
+  }
+
+  .content{
+    background-color: $white;
```

Comment:
this is temporary styles. We will change it when we implement new template styles. We will stylize all views with new css